src/index.js

```diff
@@ -1,93 +1,33 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import trimmer from 'trim-request-body';
+import cors from 'cors';
+import debug from 'debug';
 
-const isProduction = process.env.NODE_ENV === "production";
-
-// Create global app object
+// Instance of express app
 const app = express();
 
-app.use(cors());
-
-// Normal express config defaults
-app.use(require("morgan")("dev"));
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
-
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
+const infoLog = debug('http:info');
 
-app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
-);
-
-if (!isProduction) {
-    app.use(errorhandler());
-}
+// Allow cross origin access
+app.use(cors());
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
+// Parse application/json
+app.use(express.json());
 
-require("./models/User");
+// Parse application/xwww-
+app.use(express.urlencoded({ extended: false }));
 
-app.use(require("./routes"));
+// Trim the parsed request body
+app.use(trimmer);
 
-/// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-    const err = new Error("Not Found");
-    err.status = 404;
-    next(err);
+// Finally, start server...
+const server = app.listen(process.env.PORT || 3000, function() {
+    infoLog("Listening on port " + server.address().port);
 });
 
-/// error handlers
-
-// development error handler
-// will print stacktrace
-if (!isProduction) {
-    app.use(function(err, req, res, next) {
-        console.log(err.stack);
-
-        res.status(err.status || 500);
-
-        res.json({
-            errors: {
-                message: err.message,
-                error: err
-            }
-        });
-    });
-}
-
-// production error handler
-// no stacktraces leaked to user
-app.use(function(err, req, res, next) {
-    res.status(err.status || 500);
-    res.json({
-        errors: {
-            message: err.message,
-            error: {}
-        }
+app.use('*', (req, res) => {
+    res.status(200).json({
```

Comment:
This should handle route not found refactor the status code and message, and create a new middleware with the route `/` to respond with a welcome message,

Reply:
I have done that. Please check now